src/main/java/christmas/constants/EventType.java

```diff
@@ -0,0 +1,29 @@
+package christmas.constants;
+
+public enum EventType {
+    STAR("별", 5000),
+    TREE("트리", 10000),
+    SANTA("산타", 20000);
+
+    private final String name;
+    private final int nedeed;
```

Comment:
```suggestion
    private final int minimum_order_price;
```
```suggestion
    private final int required_order_price;
```
오타가 있긴한데 고치는김에 조금 더 명시적으로 적어도 될 것 같네요